spec/support/m2bf2_xquery_helpers.rb

```diff
@@ -0,0 +1,41 @@
+# helper methods for rspec
```

Comment:
By the way, if you know of a better way to run all the desired tests using a specific converter (without having to duplicate a lot of calls in all of the spec files) let me know.